codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -30,6 +31,7 @@ criticise->criticize
 criticised->criticized
 criticises->criticizes
 crystallisation->crystallization
+crystallise->crystallize
```

Comment:
crystallized, crystallizes, crystallization?

Reply:
crystallization is already in there :stuck_out_tongue_winking_eye:  But I'll add the other two :+1: